src/Adapter/Presenter/Cart/CartPresenter.php

```diff
@@ -108,6 +108,7 @@ private function presentProduct(array $rawProduct)
         $settings->allow_add_variant_to_cart_from_listing = (int) Configuration::get('PS_ATTRIBUTE_CATEGORY_DISPLAY');
         $settings->stock_management_enabled = Configuration::get('PS_STOCK_MANAGEMENT');
         $settings->showPrices = Configuration::showPrices();
+        $settings->showLabelOOSListingPages = (bool) Configuration::get('PS_SHOW_LABEL_OOS_LISTING_PAGES');
```

Comment:
It could be nice to use the configuration object here as you did in `src/Adapter/Presenter/Product/ProductLazyArray.php`, so that you can do : 
`$this->configuration->get('PS_SHOW_LABEL_OOS_LISTING_PAGES');`, 

but it's a little bit out of scope I guess.

Reply:
I approve. Out of Scope.